docs/pages/management/guides/teleport-operator.mdx

```diff
@@ -28,6 +28,12 @@ Currently supported Teleport resources are:
 - GitHub connectors
 - Login Rules
 
+<Admonition type="note">
```

Comment:
I think we can find a clearer placement for this text. For me, this location implies that it has to do with the kinds of Teleport resources we currently support, leading me to wonder why, for example, "OIDC connector" starts with an uppercase letter.

I would add this text to Step 2 instead and refer explicitly to the `metadata.name` field.